LCM/scripts/python3/SetDscLocalConfigurationManager.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+from imp                  import load_source
+from os.path              import dirname, isfile, join, realpath
+from subprocess           import PIPE, Popen
+from sys                  import argv, exc_info, exit, version_info
+from traceback            import format_exc
+from fcntl                import flock, LOCK_EX, LOCK_UN, LOCK_NB
+from OmsConfigHostHelpers import write_omsconfig_host_telemetry, write_omsconfig_host_switch_event, write_omsconfig_host_log, stop_old_host_instances
+from time                 import sleep
+
+pathToCurrentScript = realpath(__file__)
+pathToCommonScriptsFolder = dirname(pathToCurrentScript)
+
+helperLibPath = join(pathToCommonScriptsFolder, 'helperlib.py')
+helperlib = load_source('helperlib', helperLibPath)
+
+operationStatusUtilityPath = join(pathToCommonScriptsFolder, 'OperationStatusUtility.py')
+operationStatusUtility = load_source('operationStatusUtility', operationStatusUtilityPath)
+
+operation = 'SetLCM'
+
+def usage():
+    print("Usage:")
+    print("    " + argv[0] + " -configurationmof FILE")
+    write_omsconfig_host_log('Incorrect parameters to SetDscLocalConfigurationManager.py: ' + str(argv), pathToCurrentScript, 'WARNING')
+    exit(1)
+
+def main(args):
+    try:
+        apply_meta_config(args)
+    except SystemExit:
+        exit(exc_info()[1])
+    except Exception:
+        # Python 2.4-2.7 and 2.6-3 recognize different formats for exceptions. This methods works in all versions.
+        formattedExceptionMessage = format_exc()
+        write_omsconfig_host_log('Python exception raised from SetDscLocalConfigurationManager.py: ' + formattedExceptionMessage, pathToCurrentScript, 'ERROR')
+        raise
+
+def apply_meta_config(args):
+    if len(args) != 3:
+        usage()
+
+    if args[1].lower() != '-configurationmof':
+        usage()
+
+    if (not isfile(args[2])):
+        errorMessage = 'The provided configurationmof file does not exist: ' + str(args[2])
+        print(errorMessage)
+        write_omsconfig_host_log('Incorrect parameters to SetDscLocalConfigurationManager.py: ' + errorMessage, pathToCurrentScript, 'ERROR')
+        exit(1)
+
+    fileHandle = open(args[2], 'r')
```

Comment:
import codecs
    fileHandle = codecs.open(args[2], 'rb')
    try:
        fileContent = fileHandle.read()
Please replace with this, as there were some issues reading the mof file. read the file as bytes and then decode.